### DISCUSSION TITLE
Where to put .d.ts file for plugin?

### DISCUSSION THREAD

**Discussion by liborjelinek** [vuejs/test-utils#2238] [General]
I am heavily using the plugin adding `findByText()` feature in [this blogpost](https://dev.to/kouts/findbytext-for-vue-test-utils-5h2e). (BTW you should add something like this to the VTU core. It is a must.)

According to [the docs on plugins](https://test-utils.vuejs.org/guide/extending-vtu/plugins.html#Using-the-plugin-with-TypeScript) I created `vue-test-utils.d.ts` with content

```
import { DOMWrapper } from '@vue/test-utils';

declare module '@vue/test-utils' {
  export class VueWrapper {
    findByText(text: string, selector = '*'): DOMWrapper[];
  }
}
```

But where to place it? My VS Code underline `findByText()` as `Property 'findByText' does not exist on type` error.

**Comment by cexbrayat** [comment]
Hi @liborjelinek 

I usually to put it at the root of the project, so maybe there is an issue.

Can you provide us a small repro online using https://stackblitz.com/github/vuejs/create-vue-templates/tree/main/typescript-vitest?file=src%2Fcomponents%2F__tests__%2FHelloWorld.spec.ts ? 

It only takes a few minutes, and we'll be able to take a look

**Reply by liborjelinek** [reply]
Okay, I confirm that placing `vue-test-utils.d.ts` to the root is enough. It is something with my local setup. Maybe I have some old dependencies.

Anyway, I am taking back my question :-)
